### PR TITLE
Update to Scala 2.12.9

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -25,7 +25,7 @@ object Dependencies {
   val jacksonVersion = "2.9.9"
   val jacksonDatabindVersion = "2.9.9.1"
 
-  val scala212Version = "2.12.8"
+  val scala212Version = "2.12.9-bin-d386f96"
```
Comment (helena) [inline project/Dependencies.scala]: Do we do that? upgrade to a `bin-*` ?

Comment (raboof) [reply project/Dependencies.scala]: This is a draft PR just to see what CI thinks ;)
We'll update and mark 'Ready for review' when 2.12.9 is actually out

**Comment by helena** [comment]
Just me or did travis stall...

**Comment by helena** [comment]
OIC it's a draft still, never mind.

**Review by jrudolph — APPROVED** [review]
LGTM

**Review by helena — APPROVED** [review]
LGTM

**Pull request merged**